cornflow-dags/tests/test_dags_af.py

```diff
@@ -30,39 +31,38 @@ def run_update_all_variables_until_finished(self):
         finished = False
         while not finished:
             time.sleep(2)
-            status = client.get_dag_run_status(
-                "update_all_schemas", data["dag_run_id"]
-            )
+            status = client.get_dag_run_status("update_all_schemas", data["dag_run_id"])
             state = status.json()["state"]
             finished = state != "running"
             print("STATUS OF update_all_variables: {}".format(state))
         return client
 
+    def test_all_dags_exist(self):
+        names = [app.name for app in self.apps]
+        missing = set(EXISTING_APPS).difference(names)
+        print("Missing apps: {}".format(missing))
+        self.assertEqual(len(missing), 0)
```

Comment:
I am not sure I see the point of this test. `EXISTING_APPS` and `names` are defined in equivalent ways, so the comparison really doesn't make sense to me